zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -523,10 +523,8 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
   $scope.aceChanged = function() {
     $scope.dirtyText = $scope.editor.getSession().getValue();
     $scope.startSaveTimer();
-
-    $timeout(function() {
-      $scope.setParagraphMode($scope.editor.getSession(), $scope.dirtyText, $scope.editor.getCursorPosition());
-    });
+    var setParagraphMode = $scope.setParagraphMode;
```

Comment:
@corneadoug 
I seem to have misunderstood your say.
Modification was completed.
Thank you.